src/Manager.jsx

```diff
@@ -11,7 +11,9 @@ class Manager extends Component {
   }
 
   static defaultProps = {
-    tag: 'div',
+    // Using React 15 or lower? Remember to set this to `div` or
+    // another valid HTML tag!
+    tag: false,
```

Comment:
I was also thinking to automatically detect the React version and set the default value accordingly, but I'm not sure it's a good idea.

Reply:
@FezVrasta IMHO this is correct, just add an note to change-log/release about this and should be good... Automatically detect the React version just to set an boolean is always an bad idea! 

Overall awesome work with this 👏